dateparser/parser.py

```diff
@@ -395,15 +395,6 @@ def _results(self):
         self._set_relative_base()
 
         time = self.time() if self.time is not None else None
-
-        if self.settings.FUZZY:
-            attr_truth_values = []
-            for attr in ['day', 'month', 'year', 'time']:
-                attr_truth_values.append(getattr(self, attr, False))
-
-            if not any(attr_truth_values):
-                raise ValueError('Nothing date like found')
```

Comment:
It looks like the purpose of fuzzy is to only return a date if all those elements can be extracted for that date. Is that not a scenario that makes sense? Or is it that it does not work? (i.e. does the test also pass if it is set to `False`?)

Reply:
Hi @Gallaecio, there are several reasons to remove this setting:

1. Is not documented.
2. Only applies to `absolute-parser`.
3. Has a confusing name ("when Fuzzy is True then raise an error if the date is not complete" does this make sense for you? The semanthics are really bad.
4. We have the `STRICT_PARSING` setting which is documented and works better as well as `REQUIRE_PARTS`, that can be used with the same purpose.